lnrpc/rpc.swagger.json

```diff
@@ -789,6 +847,14 @@
     }
   },
   "definitions": {
+    "NewAddressRequestAddressType": {
```

Comment:
hmm what version of protoc and proto-gen-go are you using?

Reply:
@cfromknecht 

```[11:14:20] i:maurycy:~> protoc --version
libprotoc 3.6.0```

I'm downgrading to 3.4.0 so there is no clutter.